front/admin/cli.ts

```diff
@@ -57,6 +58,12 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
         }),
       ]);
 
+      await VaultResource.makeDefaultForWorkspace(
+        w.id,
```

Comment:
Can we pass the `auth` object here?

Reply:
not easy as we are in the process of creating a workspace, rather pass the type objects